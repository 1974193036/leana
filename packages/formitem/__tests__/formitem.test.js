import ssyyFormitem from "../src/formitem.vue"
import { mount } from "@vue/test-utils"

describe('ssyy-formitem', () => {
  test('formitem-prop', () => {
    const wrapper = mount(ssyyFormitem)
    expect(wrapper.vm.$el).toMatchSnapshot() // 生成快照
  })
})