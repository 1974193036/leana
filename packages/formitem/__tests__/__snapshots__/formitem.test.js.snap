// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ssyy-formitem formitem-prop 1`] = `
<div>
  <label>
    
  </label>
   
  <div>
     
    <!---->
  </div>
</div>
`;
